README.md

```diff
@@ -17,8 +17,123 @@ value from data whilst ensuring it is properly governed.
+![alt text](https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTrdVMpQA6q1dlb4x3iiDP0W5OG4d3UNZUCnw&usqp=CAU "possible integration daemon configuration")
+
+#Overview
+The Postgres Database Integration Daemon Connector works by periodically making a JDBC connection to a postgres databsase instance running on postgres server. It's no actually important which database accepts the connection as 
+this connector retrieves the applicable metadata for postgres system tables provided by postgres implementation of the [information schema.](https://www.postgresql.org/docs/9.1/information-schema.html)
+
+This Integration Daemon postgres connector supports the following types of database entity and will store the relevant metatdata for each corresponding egeria
+
+* Databases
+* Schemas
+* Tables
+* Views
+* Columns
+* Primary Keys
+* Foreign Keys 
+
+# Starting the Egeria Platform
+Below is an example of the java commandline used to launch the Egeria platform that will host the Integration Daemon that is in turn hosting the 
+postgres connector instance we are about to configure.
+The point of interest when launching a platform that will host connectors is the `-Dloader.path=/home/wbittles/egeria-database-connectors/postgres-connector/libs` command line option.
+The option specifies a file system directory that must contain both the postgres JDBC driver and the postgres-connector jar file.
+````
+java -Dloader.path=/home/wbittles/egeria-database-connectors/postgres-connector/libs -Dserver.port=9443 -jar /home/wbittles/egeria/open-metadata-distribution/open-metadata-assemblies/target/egeria-2.10-SNAPSHOT-distribution/egeria-omag-2.10-SNAPSHOT/server/server-chassis-spring-2.10-SNAPSHOT.jar`
+````
+
+# Postgres Connection Details
+After creating the Egeria Integration Daemon server that will host the postgres connector, the next step is to configure the connector. 
+As typical with Egeria the we configure the postgres connector by posting a JSON configuration document to a REST API endpoint. 
+
+In this readme we'll break down a cURL request, which is based on the following command from the Postman collection.
+`Configure integration daemon server with an integration service`
+
+Below we can see the typical REST endpoint and the associated header where 
+`platformName` is the name of the server hosting the egeria platform which hosts the Integration Daemon, and
+`integrationDaemonServer` is the name of the server that is to host the postgres connector.
+````shell
+ curl --location --request POST 'https://platformName:9443/open-metadata/admin-services/users/garygeeke/servers/integrationDaemonServerName/integration-services/database-integrator' \
+--header 'Content-Type: application/json' \
+````
+
+In the above section the
+omagserverPlatformRootURL is the name of the OMAG platform that is hosting the OMAG Server that hosts the OMAS that the integration daemon is partnered with.
```

Comment:
that's a mouthful. Accurate, but may simplify to just sa 'URL of the OMAG platform' (it's reasonable to expect the reader to infer the 'that is hosting' bit?

Reply:
clarified in new PR